### PR TITLE
Added our stocks API to bitflux, chart can now pass a symbol

### PR DESCRIPTION
**Pull request by JackG188**

### CODE REVIEW

File: stockflux-bitflux/src/assets/js/data/scottstock/historic/feedAdaptor.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import d3 from 'd3';
+import fcRebind from 'd3fc-rebind';
+import getScottStockData from '../../../../../../node_modules/stockflux-core/src/services/ScottStockService';
```
Comment (oriondean) [inline stockflux-bitflux/src/assets/js/data/scottstock/historic/feedAdaptor.js]: Shouldn't this be 
`import getStockFluxData from 'stockflux-core/src/services/ScottStockService';`

File: stockflux-bitflux/src/assets/js/initialiseModel.js [hunk 2]
```diff
@@ -43,14 +45,22 @@ export default function() {
                 quandlHistoricErrorResponseFormatter,
                 null,
                 null,
-                skipWeekendsDiscontinuityProvider())
+                skipWeekendsDiscontinuityProvider()),
+            scottStock: model.data.source(
```
Comment (oriondean) [inline stockflux-bitflux/src/assets/js/initialiseModel.js]: Needs renaming

File: stockflux-chart/src/components/Chart.js [hunk 3]
```diff
@@ -1,23 +1,39 @@
...
     useEffect(() => {
-
-        chart.periodsOfDataToFetch(1200);
-        chart.proportionOfDataToDisplayByDefault(112/1200);
-        // using random generated data
         chart.run(showcaseContainer);
+        if (symbol) {
+            chart.changeScottStockProduct(symbol);
+        }
+        // eslint-disable-next-line
```
Comment (oriondean) [inline stockflux-chart/src/components/Chart.js]: Why is this disabled? Are we missing a hook dependency?

File: stockflux-core/src/services/ScottStockService.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        var params = [];
+        // defaulting data to 2016-01-01 as currently UI has no influence over dates 
+        if (start != null) {
+            params.push('/' + moment(start).format('YYYY-MM-DD'));
+        }
+        if (end != null) {
+            params.push('/' + moment(end).format('YYYY-MM-DD'));
+        }
+        // change to AWS endpoint
+        var url = 'https://nuwvo3ugc3.execute-api.eu-west-2.amazonaws.com/dev/ohlc/' + product + '/2016-01-01';
```
Comment (oriondean) [inline stockflux-core/src/services/ScottStockService.js]: Do we have a proper domain for this yet?

File: stockflux-bitflux/src/assets/js/data/scottstock/historic/feedAdaptor.js (hunk 1 above)
Comment (JackG188) [reply stockflux-bitflux/src/assets/js/data/scottstock/historic/feedAdaptor.js]: tech debt, will be fixed when I remove grunt

**Pull request merged**